instruments/newport/newportesp301.py

```diff
@@ -734,7 +733,7 @@ def units(self, newval):
             return
         if isinstance(newval, int):
             self._units = self._get_pq_unit(NewportESP301Units(int(newval)))
-        elif isinstance(newval, u.Quantity):
+        elif isinstance(newval, u.Unit):
```

Comment:
Testing for u.Quantity does not allow the next call to get the Unit. Also changed the docstring. Is this an issue when switching from `quantities` to `pint` maybe?

Reply:
It might have been an error in the original version, but you are correct to update this.